### DISCUSSION TITLE
[1 febbraio 2022 - A] Esercizio 4

### DISCUSSION THREAD

**Discussion by matypist** [sapienzastudentsnetwork/psd2223#208] [Esami]
**Esercizio 4 (3 punti)**

Usando gli assiomi dell’algebra di Boole, verificare la seguente identità:

![image](https://user-images.githubusercontent.com/69389461/211824144-a0212d1c-30b5-42df-b81f-4c0c793bf1ba.png)

**Comment by Elia-Belli** [comment]
$(a\bar b + (\overline {\bar b c + \bar a c + bc}) \oplus ac$
$(a\bar b + (\overline {c + \bar a c}) \oplus ac$
$(a\bar b +\bar c) \oplus ac$
$(a\bar b +\bar c)(\bar a + \bar c) + \overline {(a\bar b +\bar c)} ac$
$a\bar b \bar c + \bar a \bar c + \bar c + (\bar a + b)ac$
$\bar c + abc$
$(a + \bar c)(b + \bar c)(\bar c + c)$
$ab + a \bar c + \bar c b + \bar c$
$ab + \bar c$

**Comment by FeddyLix17** [comment]
corretto
